### DISCUSSION TITLE
How to add line break to output variable in azure devops

### DISCUSSION THREAD

**Discussion by kostukp96** [rfennell/AzurePipelines#1304] [General]
Hi! i am use Azure Devops pipelines. I am generated release notes and after that i am send notes to email, some time ago they were sent exactly as they were in the file "releasenotes.md" (with a line break), now they are written to the file correctly, but they are written to the output variables as a solid line, how can I add line break?

**Comment by rfennell** [comment]
The release notes file is just markdown so a newline is `n

**Reply by kostukp96** [reply]
Where should I specify this setting ?

**Reply by rfennell** [reply]
Sorry I might have misled you there. I shouldn't answer queries on my phone in a rush. Better to think a bit and not confuse the template systems with the custom javascript extension model.

I think just add blank lines within your base template. 

There is a discussion [here](https://stackoverflow.com/questions/33191744/how-to-add-new-line-in-markdown-presentation) on some options.

**Reply by kostukp96** [reply]
thanx, its work)
